src/static/css/cfpb-chart-builder.less

```diff
@@ -60,18 +60,38 @@
     .chart-label();
   }
 
+  .highcharts-label__tile-map {
+    .chart-label();
+    color: @black;
+    font-family: 'AvenirNextLTW01-Demi',Arial,sans-serif;
```

Comment:
good idea! i think i have that imported already or i can add it to use the font mixins.